__tests__/stores/tableStore.js

```diff
@@ -0,0 +1,9 @@
+jest.dontMock('../test_helper');
+
+var helper = require('../test_helper');
+
+describe('TableStore', function() {
```

Comment:
I should actually just remove all the __tests__ for this commit and add them back in later.